samtranslator/plugins/sam_plugins.py

```diff
@@ -45,13 +46,13 @@ class SamPlugins(object):
     set by the plugin. SAM translator will convert this into a nice error message and display to the user.
     """
 
-    def __init__(self, initial_plugins=None):  # type: ignore[no-untyped-def]
+    def __init__(self, initial_plugins: Optional[Any] = None) -> None:
```

Comment:
No better type than `Any`? Looking at where it's used, I see `isinstance(plugin, ServerlessAppPlugin)`.

Reply:
You're right it should be `Optional[List[BasePlugin]]`